lametro/__init__.py

```diff
@@ -16,14 +17,32 @@ class Lametro(Jurisdiction):
         "events": LametroEventScraper,
     }
 
-    legislative_sessions = []
-    for year in range(2014, 2025):
-        session = {
-            "identifier": "{}".format(year),
-            "start_date": "{}-07-01".format(year),
-            "end_date": "{}-06-30".format(year + 1),
-        }
-        legislative_sessions.append(session)
+    @property
+    def legislative_sessions():
```

Comment:
```suggestion
    def legislative_sessions(self):
```

Does the property behave correctly (i.e., doesn't need a call to `fget()`) with this change?

Reply:
Great question. What happens when we instantiate a jurisdiction? Are there side effects we wish to avoid?

Reply:
Looking at the [base Jurisdiction class](https://github.com/opencivicdata/pupa/blob/2f7847cb87ed467f7afeec3f51cc704471b679c1/pupa/scrape/jurisdiction.py#L7), I don't believe anything of major consequence would happen outside of maybe taking up a bit more space in memory whenever that `session` method in `bills.py` is called. Since every other method there would also need an instance in order to function, this is looking like this would be okay.

And aside from the test, there aren't any other spots in the scrapers that try to access `legislative_sessions` so the above snippet would be the only place where a change would be needed.

Reply:
Nice! Then instantiate away, then this should be able to come in.